src/use-swr.ts

```diff
@@ -29,11 +29,10 @@ import {
 
 const IS_SERVER = typeof window === 'undefined'
 
-// polyfill for requestIdleCallback
-const rIC = IS_SERVER
-  ? null
-  : ((window as unknown) as any)['requestIdleCallback'] ||
-    ((f: (...args: any[]) => void) => setTimeout(f, 1))
+// polyfill for requestAnimationFrame
+const rAF = IS_SERVER
+  ? () => {}
```

Comment:
conflicts here, since ts requires rAF not be null, I give a empty function as fallback